Xamarin.Forms.Platform.MacOS/Renderers/ListViewRenderer.cs

```diff
@@ -472,6 +472,11 @@ public override void MouseDown(NSEvent theEvent)
 				if (clickedRow != -1)
 					(Source as ListViewDataSource)?.OnRowClicked();
 			}
+
+			public override bool ValidateProposedFirstResponder(NSResponder responder, NSEvent forEvent)
```

Comment:
Seems this might be osx version specific, it's failing to build in in our windows machines

Reply:
@rmarinho it builds fine for me
I use vs2019 16.3.10, Xamarin.iOS and Xamarin.Mac SDK   13.6.0.12 (e3c2b40)

What verion of xamarin.mac used on your build servers?

Can be connected to 
https://github.com/xamarin/xamarin-macios/issues/4837

Reply:
Looks like fix was released in vs 16.0
https://docs.microsoft.com/en-us/xamarin/mac/release-notes/5/5.6